src/Workleap.Extensions.OpenAPI/TypedResult/ExtractSchemaTypeResultFilter.cs

```diff
@@ -121,6 +135,28 @@ internal static IEnumerable<ResponseMetadata> GetResponsesMetadata(Type returnTy
         return null;
     }
 
+    private static HashSet<int> ExtractResponseCodesFromAttributes(OperationFilterContext context)
+    {
+        HashSet<int> responseCodes = [];
+        foreach (var attribute in context.MethodInfo.CustomAttributes)
+        {
+            if (attribute.AttributeType.FullName is not (ProducesResponseTypeAttribute or SwaggerResponseTypeAttribute))
+            {
+                continue;
+            }
+
+            foreach (var argument in attribute.ConstructorArguments)
```

Comment:
I think if you don't specify any httpCode the default behavior is to set to 200, we should support that.

Reply:
It is supported, we only execute the remove 200 if we detect the usage of TypedResults signature